vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -1,3 +1,160 @@
+
+## Build
+ 
+| Command| Descriptions
+|:--|:--
+|`ent bundle build [component-name]` | Build a single component
+|`ent bundle build [mfe-1] [ms-2]...`| Build one or more named components
+|`ent bundle build --all`|Build all the components in the project
+|`ent bundle build --all-ms`|Build all microservices|
+|`ent bundle build --all-mfe`|Build all the micro frontends|
+
+#### Command Details
+`ent bundle build`: Constructs the project files based on the `entando.json` provided. It executes according to the type of stack your components are built with. For instance, a React MFE starts an npm build process. A `build` log file is generated for each component inside the .entando/logs directory of your project.
+
+## Run	
+| Command| Descriptions
+|:--|:--
+|`ent bundle run [component-name]` | Locally run your single component
+|`ent bundle run [mfe-1] [ms-2]...`| Locally run one or more named components passed as arguments 
+|`ent bundle run --all`| Locally run all the components in the bundle
+|`ent bundle run --all-ms`| Locally run the microservices in the bundle
+|`ent bundle run --all-mfe`| Locally run the MFEs in the bundle
+
+#### Command Details
+`ent bundle run`: Executes processes dependent on the component type and stack. For example, mvn spring:boot will execute for a Sping Boot microservice. All the components in the bundle will run in parallel, with a log file generated for each inside the .entando/logs directory.
+
+## Package
+| Command| Descriptions
+|:--|:--
+|`ent bundle pack`|Generate the bundle artifacts, and the bundle and MS images 
+|`ent bundle pack --org [organization]`|Generate the bundle artifacts, passing the organization name to Docker Hub|
+|`ent bundle pack --file [my-dockerfile]`|Use a custom Dockerfile for your bundle |
+
+#### Command Details
+* `ent bundle pack`: The artifacts generated for micro frontends and microservices are stored in their respective folders. Their format depends on the component type.  For instance, a React micro frontend may result in HTML, JavaScript and CSS files.  
+
+Once the artifacts are generated, Docker images for the microservices are built using the Dockerfile located in each respective folder, with the Docker build command. If the Dockerfile is missing, the `pack` command exits with failure.
+
+## Publish
+| Command| Descriptions
+|:--|:--
+|ent bundle publish| Publish your Docker images to Docker registry (default)
+|ent bundle publish --org [organization]|Publish Docker images to the Docker registry, with a specified organization |
+|ent bundle publish --registry [registry]| Publish the Docker images to your Docker registry|
+
+## ECR Bundle Commands
+Entando provides a series of `ent ecr` commands for managing bundle interactions with [the Entando Component Repository](../../docs/compose/ecr-overview.md) (ECR).
+
+| Command| Descriptions
+|:--|:--
+|`ent ecr list`| Display the list of bundles associated with the current profile|
```

Comment:
If this page is now the primary reference for the ent ecr command, I think we need to add in more of the subcommands, Unfortunately there's duplication with the ent prj command but we at least need install, install --conflict-strategy=OVERRIDE, uninstall, generate-cr, and maybe also gen-secret